cmd/porter/server.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"time"
+
+	grpc "get.porter.sh/porter/pkg/grpc"
+	"get.porter.sh/porter/pkg/porter"
+	"get.porter.sh/porter/pkg/signals"
+	"github.com/spf13/cobra"
+)
+
+func buildServerCommands(p *porter.Porter) *cobra.Command {
```

Comment:
@sgettys  mentioned calling this like 'data'
I think that's a good name. So taking lessons from `plugins` implementation - let's focus on `data` and `dataserver` - I think it could remove some ambiguity about how this is supposed to be used.
Or we can call it `operator` and `operatorserver` 
Open to suggestions too 🤷‍♀️

Reply:
The sub command is not specific to the operator.  `data` seems to ambiguous to me.  How about `api-server` or `grpc-server`?

Reply:
naming things is hard, let's go with your suggestion of `api-server` ?

Reply:
Just saw this.  I'll change it to api-server